SiteConfig: fix rendering issues (#7209)
open-cluster-management/rhacm-docs: mce_acm_integration/siteconfig/siteconfig_scale_in_worker_nodes.adoc

* You have the default templates. To get familiar with the default templates, see xref:../siteconfig/siteconfig_installation_templates.adoc#default-templates[Default set of templates]
* You have installed your cluster with the {sco}. To install a cluster with the {sco}, see xref:../siteconfig/siteconfig_install_clusters.adoc#install-clusters[Installing {sno} clusters with the {sco}]

> Complete the following steps to scale in a cluster with the {sco}:

Reviewer: Similar to my earlier comment, I recommend that you say, "Complete the steps in the following sections:"

Alternatively, we can decide to not list them because there are only 2 sections. We usually use childlinks when there are more than 2 sections.

Author: Okay, I removed this part

* <<scale-in-annotation,Adding an annotation to your worker node>>
* <<scale-in-prunemanifests,Deleting the `BareMetalHost` resource of the worker node>>

> [#scale-in-annotation]

Reviewer: I recommend that the anchor for this be updated to `add-annotation-worker-node` or something similar. Maybe even `scale-add-annotation`

== Adding an annotation to your worker node

Add an annotation to your worker node for removal.

Complete the following steps to annotate worker node from the managed cluster:

> . Add an annotation in the `extraAnnotations` field of the worker node entry in the `ClusterInstance` custom resource that was used to provision your cluster:

Reviewer: Making this statement present tense
```suggestion
. Add an annotation in the `extraAnnotations` field of the worker node entry in the `ClusterInstance` custom resource that is used to provision your cluster:
```

.. If you are using GitOps ZTP, push to your Git repository and wait for Argo CD to synchronize the changes.

. Verify that the annotation is applied to the `BaremetalHost` worker resource by running the following command on the hub cluster:

+
[source,terminal]
----

> oc get bmh -n <clusterinstance_namespace>  worker-node2.example.com -ojsonpath='{.metadata.annotations}' | jq

Reviewer: seems like there is an extra space before `worker-node2.example.com`. is that intentional?

Author: I believe it's intentional but @sakhoury please confirm!

Reviewer 2: The additional space is not required - good catch @dockerymick!

oc get bmh -n <managed_cluster_namespace> --watch
oc get bmh -n <clusterinstance_namespace>  worker-node2.example.com -ojsonpath='{.metadata.annotations}' | jq
----

+
See the following example output for successful worker node deletion:

+

> [source,terminal]

Reviewer: Is there a reason why this was removed? I also recommend that the plus sign be added to help with alignment

Author: It looks like I removed it because I forgot to add `[source,terminal]` before the output. Otherwise, it was just rephrased a bit to fit the step better

  "bmac.agent-install.openshift.io/role": "master",
  "inspect.metal3.io": "disabled",
  "siteconfig.open-cluster-management.io/sync-wave": "1",
}
----

> [#scale-in-prunemanifests]

Reviewer: I recommend changing this anchor to `delete-baremetal-host-resource` or something similar

== Deleting the `BareMetalHost` resource of the worker node

Delete the `BareMetalHost` resource of the worker node that you want to be removed.

Complete the following steps to remove a worker node from the managed cluster:

> . Update the node object that you want to delete in your existing `ClusterInstance` CR with the following configuration:

Reviewer: ```suggestion
. Update the node object that you want to delete in your existing `ClusterInstance` custom resource with the following configuration:
```

.. If you are using GitOps ZTP, push to your Git repository and wait for Argo CD to synchronize the changes.

. Verify that the `BareMetalHost` resources are removed by running the following command on the hub cluster:

+
[source,terminal]
----
oc get bmh -n <clusterinstance_namespace> --watch --kubeconfig <hub_cluster_kubeconfig_filename>
----

+
See the following example output:

+
[source,terminal]
----
NAME                        STATE                        CONSUMER         ONLINE   ERROR   AGE
master-node1.example.com    provisioned                  true             81m
worker-node2.example.com    deprovisioning               true             44m
worker-node2.example.com    powering off before delete   true             20h
worker-node2.example.com    deleting                     true             50m
----

. Verify that the `Agent` resources are removed by running the following command on the hub cluster:

+
[source,terminal]
----

> oc get agents -n <clusterinstance_namespace> --watch --kubeconfig <hub_cluster_kubeconfig_filename>

Reviewer 2: ```suggestion
oc get agents -n <clusterinstance_namespace> --kubeconfig <hub_cluster_kubeconfig_filename>
```
We can do without the `watch` here since we ask that the user verifies the removal of the agent corresponding to `worker-node2.example.com` by observing that it does not exist in the output of the command.
